fastify.js

```diff
@@ -199,9 +203,13 @@ function build (options) {
 
     req.log.info({ req }, 'incoming request')
 
-    res._startTime = now()
-    res.on('finish', onResFinished)
-    res.on('error', onResFinished)
+    if (isAbstract === false) {
+      res._startTime = now()
+    }
```

Comment:
This _might_ create some deopts and prevent some inlining if logging is indeed enable (which will be the majority of cases). Can you test if the throughput drops with logging enabled?

I fear that might be the case, as I've seen this happening when properties are added within an `if`.

Reply:
I haven't seen any drops.

*NB:* In the whole application `isAbstract` is a constant

Reply:
done